demos/imagenet_classification_demo/README.md

```diff
@@ -0,0 +1,122 @@
+# Imagenet Classification C++ Demo
+
+The demo shows an example of using neural networks for image classification.
+
+You can use the following pre-trained models with the demo:
+
+* `resnet-50`
+* `vgg19`
+* `googlenet-v4`
+* all other classification models (please, check models.lst file)
+
+For more information about the pre-trained models, refer to the [model documentation](../../models/public/index.md).
+
+## How It Works
+
+On the start-up, the application reads command line parameters and loads one network to the Inference Engine for execution. Upon getting an image, it performs inference of classification and places the processed image on grid.
+
+The demo starts in "Testing mode" with fixed grid size. After calculating the average FPS result, it will switch to
+normal mode and grid will be readjusted depending on model performance. Bigger grid means higher performance.
+
+The text above each image shows whether the classification was correct: green means right class prediction, red means wrong.
+
+Use "C", "D" and "M" keys to toggle resource motitors.
```

Comment:
Adding other monitors will require updating that list. I suggest just avoiding introducing the list.